### DISCUSSION TITLE
Move Homer PGSQL to new location on Centos ?

### DISCUSSION THREAD

**Discussion by CorrieM** [sipcapture/homer#505] [Q&A] [answered]
Hi 
Please assist as pssql is not a default Postgres installation in Homer Packages, How can I move the pssql from /var/ location to / root location as I have more space on that partition for Homer.

**Comment by adubovikov** [comment, marked as answer]
https://dba.stackexchange.com/questions/28926/moving-postgresql-data-to-different-drive

https://www.digitalocean.com/community/tutorials/how-to-move-a-postgresql-data-directory-to-a-new-location-on-ubuntu-16-04

**Reply by CorrieM** [reply]
![image](https://user-images.githubusercontent.com/6371741/141981469-741714b5-47fa-441e-b6cd-071247bdcd59.png)

**Reply by CorrieM** [reply]
This is what I get when I try to stop postgresql

**Reply by adubovikov** [reply]
@CorrieM sorry, all information already provided in the links from above. Please read how to install and configure pgsql

**Reply by CorrieM** [reply]
![image](https://user-images.githubusercontent.com/6371741/141983588-2d042079-d2f9-4922-a348-90ffabbcd02d.png)
 Thanks I managed to stop it and moved the database to new partition, after I found out I need to specify the version as the service of postgresql.
